util/rom-ext-manifest-generator.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import copy
```

Comment:
Thanks for getting rid of the `deepcopy` stuff. This is now a dangling import (you can catch that sort of thing easily by running `flake8` on the file).

Reply:
> This is now a dangling import (you can catch that sort of thing easily by running flake8 on the file

You are right, I ran the main script with `lintpy` but forgot about the `c.py`

Reply:
It seems that `./util/lintpy.py --fix -f ./util/rom-ext-manifest-generator.py` doesn't catch this issue. Is the script outdated?

Reply:
Done.

Reply:
By default `lintpy.py` runs `yapf` and `isort`. Unfortunately, these aren't brilliant linters :-( You can get it to run flake8 with:
```
util/lintpy.py --tools flake8 -f ./util/rom-ext-manifest-generator.py
```
and we actually do this in CI (and don't run the other tools). Maybe the default tools used by `lintpy.py` should change.

Probably easier is just to run flake8 (`flake8 util/rom-ext-manifest-generator.py`) manually or, better, set it up in your editor. Emacs python-mode does this by default, I think. I'm not sure how ~~lesser~~ other editors work. :-)